config/webpack.config.js

```diff
@@ -21,6 +21,7 @@ const entryFiles = {
   '1995-edu-benefits': './src/js/edu-benefits/1995/edu-benefits-entry.jsx',
   '5490-edu-benefits': './src/js/edu-benefits/5490/edu-benefits-entry.jsx',
   '5495-edu-benefits': './src/js/edu-benefits/5495/edu-benefits-entry.jsx',
+  '526-claims-increase': './src/js/526/form-entry.jsx',
```

Comment:
I think there a bunch of forms under the 526 umbrella, right? I wonder if we should put this form in a sub folder with the expectation that there will be more forms that live alongside it.

Reply:
Yeah, to be honest, I don't know how this all works out. This is the 21-526EZ, which to my understanding isn't for a claims increase (that's a 526b, I think), but we're using it for a claims increase anyhow? I don't know what I'm doing. :confused:

You're right, though. I think I'll make a `src/js/disability-benefits/526EZ/` and move this all in there. Good call. :+1: